src/content/en/updates/2020/12/webassembly.md

```diff
@@ -0,0 +1,434 @@
+
+From here, we can step-in or step-over C++ statements, and observe how
+other variables are changing too:
+
+![Tooltips and Scope view showing values of `color`, `point` and other variables](/web/updates/images/2020/12/webassembly/image8.png)
+
+Okay, so this all works great when a debug information is available, but
+what if we want to debug a code that wasn’t built with the debugging
+options?
+
+## Raw WebAssembly debugging {: #raw }
+
+For example, we asked Emscripten to provide a prebuilt SDL library for
+us, instead of compiling it ourselves from the source, so – at least
+currently – there’s no way for the debugger to find associated sources.
+Let’s step-in again to get into the `SDL_RenderDrawColor`:
+
+![DevTools showing disassembly view of `mandelbrot.wasm`](/web/updates/images/2020/12/webassembly/image1.png)
+
+We’re back to the raw WebAssembly debugging experience.
+
+Now, it looks a bit scary and isn’t something most Web developers will
+ever need to deal with, but occasionally you might want to debug a
+library built without debug information – whether because it’s a
+3<sup>rd</sup>-party library you have no control over, or because you’re
+running into one of those bugs that occurs only on production.
+
+To aid in those cases, we’ve made some improvements to the basic
+debugging experience, too.
+
+First of all, if you used raw WebAssembly debugging before, you might
+notice that the entire disassembly is now shown in a single file – no
+more guessing which function a **Sources** entry `wasm-53834e3e/
+wasm-53834e3e-7` possibly corresponds to.
+
+### New name generation scheme {: #names }
+
+We improved names in the disassembly view, too. Previously you’d see
+just numeric indices, or, in case of functions, no name at all.
+
+Now we’re generating names similarly to other disassembly tools, by
+using hints from the [WebAssembly name
+section](https://webassembly.github.io/spec/core/appendix/custom.html#name-section),
+import/export paths and, finally, if everything else fails, generating
+them based on the type and the index of the item like `$func123`. You can
+see how, in the screenshot above, this already helps to get slightly
+more readable stacktraces and disassembly.
+
+When there is no type information available, it might be hard to inspect
+any values besides the primitives – for example, pointers will show up
+as regular integers, with no way of knowing what’s stored behind them in
+memory.
+
+### Memory inspection {: #memory-inspector }
+
+Previously, you could only expand the WebAssembly memory object –
+represented by `env.memory` in the **Scope** view – to look up
+individual bytes. This worked in some trivial scenarios, but wasn’t
+particularly convenient to expand and didn’t allow to reinterpret data
+in formats other than byte values. We’ve added a new feature to help
+with this, too: a linear memory inspector.
```

Comment:
Bold it

Reply:
It's not a name of the section / pane though, just a description of the feature. The pane is called "Memory Inspector", I could use that, but then "linear" would be out of place.

Reply:
Oh, I always thought "Linear memory inspector" is the panel name. Erm.. I think we should mention the panel name "Memory inspector" somewhere in this article. Probably rephrase the sentence here or mention that in the step?

Reply:
Mentioned in the step below, let me know what you think.